example/spec/parallel/kinesisTests/KinesisReplaySpec.js

```diff
@@ -91,7 +91,7 @@ describe('The Kinesis Replay API', () => {
     testConfig.streamName = streamName;
 
     const createRecord = (identifier) => ({
-      provider: `SWOT_PODAAC${testSuffix}`,
+      provider: `PODAAC_SWOT${testSuffix}`,
```

Comment:
While I was relying on the provider in the message, I found out that this provider ID is incorrect and does not match the fixture. I've since reverted the logic to rely on that field, but left in the change to the correct provider. So, it briefly was, but isn't.